src/mainframe/models/schemas.py

```diff
@@ -16,6 +19,60 @@ class Error(BaseModel):
     detail: str
 
 
+class Package(BaseModel):
+    """Model representing a package queried from the database."""
+
+    scan_id: str
+    name: str
+    version: str = ""
+    status: str = ""
+    score: Optional[int]
```

Comment:
Perhaps I'm being dumb, but I don't see why any of the `Optional` fields can't consistently have a `None` default, or the opposite (not having a default whatsoever), since `Scan` provides defaults for all `Optional` fields anyways.